### ISSUE TITLE
About the train-accuracy between mxnet and caffe

### ISSUE THREAD

**Issue by idealboy** [apache/incubator-mxnet#7592]
For bugs or installation issues, please provide the following information.
The more information you provide, the more likely people will be able to help you.

## Environment info
Operating System:
cenos 7.2
Compiler:
gcc4.8.5
Package used (Python/R/Scala/Julia):
python
MXNet version:
0.9.5
Or if installed from source:

MXNet commit hash (`git rev-parse HEAD`):

If you are using python package, please provide
python-2.7
Python version and distribution:
python-2.7
If you are using R package, please provide

R `sessionInfo()`:

## Error Message:
Please paste the full error message, including stack trace.

when I train my resnet-50 model on caffe, it achieves about 97% for the train-accuracy, but with the same training examples and the same parameter setting(batch_size:32, lr:0.01, momentum:0.9, weight decay: 0.0002, epoch:30, step:10,20, respectly, the caffe has the same setting), it only achieve about 93% for the train-accuracy, thank you for your help!

## Minimum reproducible example
if you are using your own code, please provide a short script that reproduces the error.

## Steps to reproduce
or if you are running standard examples, please provide the commands you have run that lead to the error.

1.
2.
3.

## What have you tried to solve it?

1.
2.
3.

**Comment by idealboy** [comment]
when I train my resnet-50 model on caffe, it achieves about 97% for the train-accuracy, but with the same training examples and the same parameter setting(batch_size:32, lr:0.01, momentum:0.9, weight decay: 0.0002, epoch:30, step:10,20, respectly, the caffe has the same setting), it only achieve about 93% for the train-accuracy, thank you for your help!

**Comment by tornadomeet** [comment]
what's your data augmentation in caffe and mxnet?

**Comment by idealboy** [comment]
In caffe, only "mirror" is specified in the prototxt configuration, when in trainning, the image was resize to 256*256, and then random crop to 224*224

In mxnet, I use the default setting in the data augumentation, and also 256*256 to 224*224 by random crop as I know

I want to use CaffeOP to transfer the original caffe network to mxnet, but I don't know whether it is effective, Now I am doing this task!

**Comment by idealboy** [comment]
@tornadomeet 

Thank you for your attention for this problem!

In caffe, only "mirror" is specified in the prototxt configuration, when in trainning, the image was resize to 256256, and then random crop to 224224

In mxnet, I use the default setting in the data augumentation, and also 256256 to 224224 by random crop as I know


I want to use CaffeOP to transfer the original caffe network to mxnet, but I don't know whether it is effective, Now I am doing this task!

**Comment by mnikitin** [comment]
@idealboy 

Have you figured out what was wrong? I have the same problem, my caffe-trained resnet performs better than mxnet-trained, with the same train settings.
Thanks!

**Comment by szha** [comment]
@apache/mxnet-committers: This issue has been inactive for the past 90 days. It has no label and needs triage.

For general "how-to" questions, our [user forum](https://discuss.mxnet.io/) (and [Chinese version](https://discuss.gluon.ai/)) is a good place to get help.

**Comment by SuperLinguini** [comment]
Proposed Label: Performance, Python

**Comment by lupesko** [comment]
@idealboy this issue has been open with no activity for over a year now.
Is it still relevant?
If it is - please add more context and details, including training code in MXNet and Caffe, so the community can try and help. Otherwise - I suggest you close the issue. Thanks!

**Comment by sandeep-krishnamurthy** [comment]
With Gluon-CV examples, we see similar to better accuracy reached with MXNet. Closing this issues due to inactivity and no reproducible samples.
Please reopen if issue still persists and need help.
